### PR TITLE
Fixed inconsistent button behaviour on hover

### LINKED ISSUE #3757: [UI] Inconsistent button behavior on hover
### Description
The following video shows the issue: 

https://user-images.githubusercontent.com/75200954/218549292-ac76a7e1-4c57-4f12-b3b2-743553d375fd.mp4

### Expected Behavior

Basically, the button should have box shadow on hover and text should be black.

### Environment:
- Host OS: 
- Browser: 

---
<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
- 🎨 Wireframes and designs for Layer5 site in [Figma](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). 
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io).

Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

### PR DESCRIPTION
**Pull request by AkashS20**
**Description**

This PR fixes #3757 

[video](https://user-images.githubusercontent.com/92908347/230170155-7ba0bf78-4e8b-48a8-805e-2428d6ae21c9.webm)

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit c242fba0250bd3017f49321ae373291fe5de90b7 at: https://642dc19e0ef1950f5a9c1c56--layer5.netlify.app

File: src/sections/Meshery/Meshery-integrations/index.js [hunk 1]
```diff
@@ -25,7 +25,7 @@ const Integrations = ({ theme }) => {
               <>
                 <IntegrationsGrid theme={theme} count={13} />
                 <div style={{ marginTop: "20px" }}>
-                  <Button
+                  <Button className="seeAll-btn"
```
Comment (Udit-takkar) [inline src/sections/Meshery/Meshery-integrations/index.js:28]: Please use consistent name of class like see-all-btn or seeAllBtn

Comment (AkashS20) [reply src/sections/Meshery/Meshery-integrations/index.js]: I have made the necessary changes

**Comment by l5io** [comment]
🚀 Preview for commit d5bbf791230b564f3244d721c3b578e38ef03e97 at: https://642e130a9b78fa2aa17c19d9--layer5.netlify.app

**Review by ayushthe1 — APPROVED** [review]
Thanks :+1:

**Review by Udit-takkar — APPROVED** [review]
LGTM

**Pull request merged**